sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -127,7 +127,7 @@ case class CatalogTable(
     sortColumnNames: Seq[String] = Seq.empty,
     bucketColumnNames: Seq[String] = Seq.empty,
     numBuckets: Int = -1,
-    owner: String = "",
+    owner: String = System.getProperty("user.name"),
```

Comment:
Is this definitely the owner? it doesn't seem like it would generally be the user happening to run the application.